AUTHORS

```diff
@@ -190,6 +190,7 @@ Shane Booth (shane31)
 Shawn Varghese (xXH4CKST3RXx)
 Siad Daboul (Topologist)
 Stefan Geschwentner (locutus2)
+Stefano Di Martino (StefanoD)
```

Comment:
Should be moved one line down for alphabetic order.

Reply:
Done